node/tools/src/k8s.rs

```diff
@@ -172,30 +187,32 @@ pub async fn deploy_node(
     Ok(())
 }
 
-/// Returns a HashMap with mapping: node_id -> IP address
-pub async fn get_seed_node_addrs(
+/// Waits for the pods to start in order to complete each ConsensusNode with its assigned NodeAddr
+pub async fn find_node_addrs(
```

Comment:
This should become a method which can be applied to a given `ConsensusNode` instance. Also, shouldn't be associated with the notion of seed nodes.

Reply:
On one hand, method may be useful, but on the other this way is more efficient as it queries the cluster for a set of nodes in a single request (and waits for all of them to be available and running on an assigned IP address).

If this was a method in a given `ConsensusNode` we will have to query and wait for each pod individually instead

Reply:
Optimizing for efficiency isn't very important here. (and it can be done later, if needed)
There's a need for a basic functionality in which you can `.waitPod()` on a given node until ready, to perform various operations thereafter. (obtain IP address, connect RPC client, etc.).
This is needed for all nodes, not just seed nodes.